README.md

```diff
@@ -54,9 +79,36 @@ react-native link @react-native-community/progress-view
 
 ## Usage
 
+Import ProgressView from `@react-native-community/progress-view`
+
 ```javascript
 import {ProgressView} from "@react-native-community/progress-view";
 ```
+Add ProgressView like this
+
+```javascript
+<ProgressView
+          progressTintColor="orange"
+          trackTintColor="blue"
+          progress={0.7}
+/>
+```
+
+### Running Example App
+
+#### Windows
+1. Clone branch 
+2. cd into progress_view and run `yarn install`
+3. run `yarn add react-native@0.62 --dev`
```

Comment:
I think you should update package.json instead of running "yarn add" everytime.

Reply:
I'm not sure why but when I update the package.json to add '`yarn add react-native@0.62 --dev`', it breaks pod install for iOS.

Reply:
Are you adding RNW as a devDependency? This is necessary; Kai is right, that it should be this way.

Reply:
RNW is a devDependency, i think it's because the IOS app relies on react-native": "^0.63.0-rc.1", I can try to play around with package.json and see if there's a release that works with both